app/javascript/components/admins/AdminNavbar.js

```diff
@@ -27,9 +48,30 @@ class AdminNavbar extends React.Component {
           title="Profile"
         >
           <div className="pt-dialog-body">
-            <h6>Name</h6> {this.props.admin.first_name + " " + this.props.admin.last_name}
-            <h6>Email</h6> {this.props.admin.email}
-            <h6>Description</h6> {this.props.admin.description}
+            <h6>First Name</h6>
+            <EditableText
+              defaultValue = {this.state.values[first_name]}
+              // disabled = true
+              onChange = {event => this.handleChange(first_name, event.target.value)}
+              />
+            <h6>Last Name</h6>
+            <EditableText
+              defaultValue = {event => this.state.values[last_name]}
+              // disabled = true
+              onChange = {event => this.handleChange(last_name, event.target.value)}
+            />
+            <h6>Email</h6>
+            <EditableText
+              defaultValue = {this.state.values['email']}
+              // disabled = true
+              onChange = {event => this.handleChange(email, event.target.value)}
+            />
+            <h6>Description</h6>
+            <EditableText
+              defaultValue = {this.state.values['description']}
+              // disabled = true
+              onChange = {this.handleChange(description, event.target.value)}
```

Comment:
This is the issue - this is calling handleChange repetitively, you either need it to be like how you have above (i.e. `onChange = {event => this.handleChange(description, event.target.value)}` or you can do it similar to how its done here https://github.com/calblueprint/veterans360-web/blob/master/app/javascript/components/partnering_organizations/ViewResources.js#L82